src/core/server/ui_settings/create_or_upgrade_saved_config/create_or_upgrade_saved_config.ts

```diff
@@ -63,7 +63,12 @@ export async function createOrUpgradeSavedConfig(
 
   try {
     // create the new SavedConfig
-    await savedObjectsClient.create('config', attributes, { id: version });
+    await savedObjectsClient.create('config', attributes, {
+      id: version,
+      // config is a global object that should not belong to any of the workspaces
+      // declare it as null to prevent it being created as a workspace object
+      workspaces: null,
+    });
```

Comment:
I am thinking of whether we can do the same as `namespaceType` to add a new type for workspace and don't add workspace options in wrapper, so that we don't need to add this logic for each fix id saved object.

Reply:
Either adding a `workspaceType` or adding field `workspaces: null` should work. And it is more flexible by using latter option. Furthermore if we take workspace-level settings into consideration, they may use the same type.